third_party/terraform/data_sources/data_source_google_projects.go

```diff
@@ -79,11 +110,38 @@ func flattenDatasourceGoogleProjectsList(v interface{}) []map[string]interface{}
 	projects := make([]map[string]interface{}, 0, len(ls))
 	for _, raw := range ls {
 		p := raw.(map[string]interface{})
+
+		var mId, mNumber, mLabels, mLifecycleState, mName, mCreateTime, mParent interface{}
 		if pId, ok := p["projectId"]; ok {
-			projects = append(projects, map[string]interface{}{
-				"project_id": pId,
-			})
+			mId = pId
+		}
+		if pNumber, ok := p["projectNumber"]; ok {
+			mNumber = pNumber
+		}
+		if pName, ok := p["name"]; ok {
+			mName = pName
+		}
+		if pLabels, ok := p["labels"]; ok {
+			mLabels = pLabels
+		}
+		if pLifecycleState, ok := p["lifecycleState"]; ok {
+			mLifecycleState = pLifecycleState
+		}
+		if pCreateTime, ok := p["createTime"]; ok {
+			mCreateTime = pCreateTime
+		}
+		if pParent, ok := p["parent"]; ok {
+			mParent = pParent
```

Comment:
I think something is probably a little mixed up about this typing - if you want to investigate, I'd log the value you're receiving, including type, at p["parent"].

Reply:
I figured it out, was testing the wrong fields. parent instead parent.id and parent.type.